pkgs/data/themes/pop-gtk/default.nix

```
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchurl
+, intltool
+, bash
+, which
+, parallel
+, pkgconfig
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "3.1.2";
+
+  src = fetchurl {
+    url    = "https://github.com/pop-os/gtk-theme/archive/${version}.tar.gz";
+    sha256 = "15pj5rhhpn3clb87973m9bbmp02z4rv0d88h0idmpdg4i0yj5q8a";
+  };
+
+  nativeBuildInputs = [
+    bash
+    which
+    parallel
+    intltool
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  propagatedUserEnvPkgs = [
+    gnome3.gnome-themes-extra
+    gtk-engine-murrine
+  ];
+
+  preBuild = ''
+    export HOME="$NIX_BUILD_ROOT"
+
+    patchShebangs .
+
+    substituteInPlace Makefile \
+      --replace '/bin/bash' '${bash}/bin/bash'
+
+    substituteInPlace Makefile \
+      --replace '/usr/share/themes' '${placeholder "out"}/share/themes'
+
+    substituteInPlace src/gtk-2.0/render-asset.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-2.0/render-asset.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
+
+    substituteInPlace src/gtk-2.0/render-assets-dark.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-2.0/render-assets-dark.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
+
+    substituteInPlace src/gtk-3.0/gtk-common/render-asset.sh \
+      --replace '/usr/bin/inkscape' '${inkscape}/bin/inkscape'
+
+    substituteInPlace src/gtk-3.0/gtk-common/render-asset.sh \
+      --replace '/usr/bin/optipng' '${optipng}/bin/optipng'
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "System76 Pop GTK+ Theme";
+    homepage    = https://github.com/pop-os/gtk-theme;
```

```suggestion
    homepage    = "https://github.com/pop-os/gtk-theme";
```

We had a [recent decision](https://github.com/NixOS/rfcs/pull/45) to favor quoting URLs. It's not yet generalized in nixpkgs but since this is an init of a new pkg, let's do it right from the start. :)